fortuna/data/loader/huggingface_loaders.py

```diff
@@ -0,0 +1,53 @@
+from typing import Union, Dict, Iterable, Optional, Tuple
+from jax import numpy as jnp
+
+from fortuna.data.loader.base import BaseDataLoaderABC, BaseInputsLoader, BaseTargetsLoader
+from fortuna.data.loader.utils import IterableData
+from fortuna.typing import Array
+
+
+class HuggingFaceDataLoader(BaseDataLoaderABC):
+
+    def __init__(
+            self,
+            iterable: Union[Iterable[Dict[str, Array]], Iterable[Tuple[Dict[str, Array], Array]]],
+            num_unique_labels: int = None,
+            num_input_samples: Optional[int] = None
```

Comment:
Can you add these two arguments to the docstring?

Also, what do you mean. by samples? If this is just the number of inputs, I'd just call it `num_inputs`. I'd prefer avoiding the confusion of what a "sample" is.